components/base/agilex/limo_base_test.go

```diff
@@ -7,28 +7,30 @@ import (
 	"github.com/edaniels/golog"
 	"go.viam.com/test"
 
+	"go.viam.com/rdk/components/base"
 	"go.viam.com/rdk/resource"
 )
 
 func TestLimoBaseConstructor(t *testing.T) {
 	ctx := context.Background()
 	logger := golog.NewTestLogger(t)
+	deps := resource.Dependencies{}
 
 	c := make(chan []uint8, 100)
 
-	_, err := CreateLimoBase(context.Background(), resource.Config{ConvertedAttributes: &Config{}}, logger)
+	_, err := createLimoBase(context.Background(), deps, resource.Config{ConvertedAttributes: &Config{}}, logger)
 	test.That(t, err, test.ShouldNotBeNil)
 
 	cfg := &Config{
 		DriveMode: "ackermann",
 		TestChan:  c,
 	}
 
-	baseBase, err := CreateLimoBase(context.Background(), resource.Config{ConvertedAttributes: cfg}, logger)
+	lb, err := createLimoBase(context.Background(), deps, resource.Config{ConvertedAttributes: cfg}, logger)
 	test.That(t, err, test.ShouldBeNil)
-	base, ok := baseBase.(*limoBase)
-	test.That(t, ok, test.ShouldBeTrue)
-	width, _ := base.Width(ctx)
-	test.That(t, width, test.ShouldEqual, 172)
-	base.Close(ctx)
+	props, err := lb.Properties(ctx, nil)
```

Comment:
Do you want to add a test for the turning radius as well?

Reply:
+1. particularly for different steering modes